src/app/components/Billboard/index.test.tsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { fireEvent, render, screen } from '../react-testing-library-with-providers';
+import Billboard from '.';
+import { kyrgyzBillboard } from './fixtures';
+import * as viewTracking from '../../hooks/useViewTracker';
+import * as clickTracking from '../../hooks/useClickTrackerHandler';
+
+describe('Billboard', () => {
+  const summary = kyrgyzBillboard.summaries[0];
+  const eventTrackingData = { componentName: 'billboard' };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('for a curation with 1 summary', () => {
+    it('should render a section with role region', () => {
+      render(
+        <Billboard
+          heading={kyrgyzBillboard.title}
+          description={summary.description}
+          link={summary.link}
+          image={summary.imageUrl}
+          eventTrackingData={eventTrackingData}
+          showLiveLabel
```

Comment:
I think we should have a separate test for the present of the live label in the billboard, and have all other tests without it

```suggestion
```